packages/itwinui-css/src/field.scss

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+:where(.iui-field) {
+  --_iui-field-color-text: var(--iui-color-text);
+  --_iui-field-color-text-hover: var(--iui-color-text-hover);
+  --_iui-field-color-text-disabled: var(--iui-color-text-disabled);
+  --_iui-field-color-background: var(--iui-color-background);
+  --_iui-field-color-background-hover: var(--iui-color-background-hover);
+  --_iui-field-color-background-disabled: var(--iui-color-background-disabled);
+  --_iui-field-color-border: var(--iui-color-border-foreground);
+  --_iui-field-color-border-hover: var(--iui-color-border-foreground-hover);
+  --_iui-field-color-border-disabled: var(--iui-color-border-disabled);
+  --_iui-field-color-focus-outline: var(--iui-color-border-accent);
+  --_iui-field-height: var(--iui-component-height);
+  --_iui-field-focus-outline-offset: -2px;
+  --_iui-field-focus-outline-width: 2px;
+  --_iui-field-font-size: var(--iui-font-size-1);
+
+  background-color: var(--_iui-field-color-background);
+  border: 1px solid var(--_iui-field-color-border);
+  border-radius: var(--iui-border-radius-1);
+  color: var(--_iui-field-color-text);
+  font-family: inherit;
+  font-size: var(--_iui-field-font-size);
+  font-weight: var(--iui-font-weight-normal);
+  min-block-size: var(--_iui-field-height);
+  min-inline-size: var(--_iui-field-height);
+  -webkit-tap-highlight-color: var(--_iui-field-color-background-hover);
+  transition:
+    background-color var(--iui-duration-1) ease-out,
+    border-color var(--iui-duration-1) ease-out,
+    color var(--iui-duration-1) ease-out;
+
+  &:where(:hover) {
+    background-color: var(--_iui-field-color-background-hover);
+    border-color: var(--_iui-field-color-border-hover);
+    color: var(--_iui-field-color-text-hover);
+  }
+
+  &:where(:active) {
+    background-color: var(--_iui-field-color-background-hover);
+    border-color: var(--_iui-field-color-border-hover);
+    color: var(--_iui-field-color-text-hover);
+  }
+
+  &:where(:focus-visible) {
+    outline: var(--_iui-field-focus-outline-width) solid var(--_iui-field-color-focus-outline);
+    outline-offset: var(--_iui-field-focus-outline-offset);
+  }
+
+  &:where([disabled], :disabled, [aria-disabled='true'], [data-iui-disabled='true'], .iui-disabled) {
```

Comment:
Same question about `iui-disabled`

Reply:
Looks like `select` is using `.iui-disabled` so needs to stay in `.iui-field` unless we were to update `select`.

Reply:
Can you we change that to also use `data-iui-disabled`?

Reply:
Done in https://github.com/iTwin/iTwinUI/pull/1919/commits/fb546c8c636fe0c67e3a0b217f8fed07f3f29adb, @rohan-kadkol could you take care of the React side for this please?

Reply:
Absolutely, done (https://github.com/iTwin/iTwinUI/pull/1919/commits/88233c00a5b21b27ed7b7c237456c4750a36b9d3).